db/migrate/20250325145324_assign_customers_to_billing_entities.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AssignCustomersToBillingEntities < ActiveRecord::Migration[7.2]
+  def change
+    # NOTE: ensure first billing entity has the same id as the organization to ease the migration to multi entities.
+    BillingEntity.where("id != organization_id").find_in_batches(batch_size: 1000) do |batch|
+      BillingEntity.where(id: batch.pluck(:id))
+        .update_all("id = organization_id") # rubocop:disable Rails/SkipsModelValidations
+    end
+
+    # NOTE: Update all customers to have the same billing_entity_id as organization_id
+    Customer.where("billing_entity_id != organization_id").or(Customer.where(billing_entity_id: nil)).find_in_batches(batch_size: 1000) do |batch|
```

Comment:
I think we can omit 
`.or(Customer.where(billing_entity_id: nil))`, because if billing_entity_id is nil, it will not be equal to organization_id

Reply:
we need it, SQL follows three-valued logic (true, false, and unknown). The key reason rows where billing_entity_id is NULL and organization_id is not NULL are excluded is that:

* Any comparison involving NULL results in unknown (not true or false).
* The WHERE clause only includes rows where the condition evaluates to true.
* Since NULL != b evaluates to unknown, those rows are filtered out.